Library/ViewModels/ProjectSummaryCarouselCellViewModel.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveSwift
+
+public protocol ProjectSummaryCarouselCellViewModelInputs {
+  func configure(with item: ProjectSummaryEnvelope.ProjectSummaryItem)
+}
+
+public protocol ProjectSummaryCarouselCellViewModelOutputs {
+  var body: Signal<String, Never> { get }
+  var title: Signal<String, Never> { get }
+}
+
+public protocol ProjectSummaryCarouselCellViewModelType {
+  var inputs: ProjectSummaryCarouselCellViewModelInputs { get }
+  var outputs: ProjectSummaryCarouselCellViewModelOutputs { get }
+}
+
+public final class ProjectSummaryCarouselCellViewModel: ProjectSummaryCarouselCellViewModelType,
+  ProjectSummaryCarouselCellViewModelInputs, ProjectSummaryCarouselCellViewModelOutputs {
+  public init() {
+    let item = self.itemProperty.signal
+      .skipNil()
+
+    self.body = item
+      .map(\.response)
+
+    self.title = item
+      .map(\.question)
+      .map(ProjectSummaryCarouselCellViewModel.titleText(for:))
+  }
+
+  private let itemProperty = MutableProperty<ProjectSummaryEnvelope.ProjectSummaryItem?>(nil)
```

Comment:
Just a suggesting but I feel like usually we prefix the name of these properties with `configureWith` - so this would be `configureWithItemProperty` 🤷‍♀